dev-python/ordered-set/ordered-set-3.1.1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A mutable set that remembers the order of its entries"
+HOMEPAGE="https://github.com/LuminosoInsight/ordered-set"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+distutils_enable_tests pytest
+
+python_test() {
+	pytest -vv test.py || die "Tests fail with ${EPYTHON}"
+}
```

Comment:
Won't this work without defining it, just with `distutils_enable_tests pytest`?

Reply:
Nope, sadly not,

The pypi release tarball is missing the `pytest.ini` file that specifies which file to run, that's why I do it like this.